update.yaml

```diff
@@ -0,0 +1,8 @@
+version: 1.0.0
+title: New Stable release
```

Comment:
@Saviq are you planning to update this to real information before we merge it?

Reply:
@nottrobin yes, provided there's agreement on the format and approach :)

Reply:
I think as a first pass the format is fine, and @anthonydillon appears to agree based on his approval, so please go ahead and make this real info and we can merge whenever you're ready.

Reply:
@nottrobin done.